x-pack/plugins/rule_registry/common/assets/field_maps/ecs_field_map.ts

```diff
@@ -1110,6 +1495,21 @@ export const ecsFieldMap = {
     array: false,
     required: false,
   },
+  'host.cpu.usage': {
+    type: 'scaled_float',
+    array: false,
+    required: false,
+  },
```

Comment:
Looks like this is the source of the mapping error, the script that generates this file doesn't pick up the `scaling_factor` from the ECS file (https://github.com/elastic/ecs/blob/main/generated/ecs/ecs_flat.yml#L4646) and `scaling_factor` is a required property for `scaled_float` fields.